server/service/StringProfileService.js

```diff
@@ -79,8 +79,8 @@ exports.putStringProfileStringValue = async function (url, body) {
   return new Promise(async function (resolve, reject) {
     try {
       let stringValue = body["string-profile-1-0:string-value"]
-      let stringProfileList = await stringProfile.getStringProfile('aa-2-1-2-string-p-001')
-      let stringProfilePattern = stringProfileList["stringProfilePac"]["stringProfileCapability"]['pattern']
+      let stringProfileList = await stringProfile.getStringValueAndPattern('NameOfOperationForInquiringApprovals')
```

Comment:
For the putStringProfileStringValue , would'nt the uuid will be there as a part parameter ? Please get the uuid from the path parameter.